src/main/java/io/github/nucleuspowered/nucleus/modules/misc/commands/ItemInfoCommand.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * This file is part of Nucleus, licensed under the MIT License (MIT). See the LICENSE.txt file
+ * at the root of this project for more details.
+ */
+package io.github.nucleuspowered.nucleus.modules.misc.commands;
+
+import io.github.nucleuspowered.nucleus.Util;
+import io.github.nucleuspowered.nucleus.internal.CommandBase;
+import io.github.nucleuspowered.nucleus.internal.annotations.Permissions;
+import io.github.nucleuspowered.nucleus.internal.annotations.RegisterCommand;
+import org.spongepowered.api.command.CommandResult;
+import org.spongepowered.api.command.args.CommandContext;
+import org.spongepowered.api.command.spec.CommandSpec;
+import org.spongepowered.api.entity.living.player.Player;
+import org.spongepowered.api.item.inventory.ItemStack;
+
+@Permissions
+@RegisterCommand({"iteminfo", "itemdb"})
+public class ItemInfoCommand extends CommandBase<Player> {
+
+    @Override
+    public CommandSpec createSpec() {
+        return CommandSpec.builder().executor(this).build();
+    }
+
+    @Override
+    public CommandResult executeCommand(Player player, CommandContext args) throws Exception {
+        if (player.getItemInHand().isPresent()) {
+            ItemStack it = player.getItemInHand().get();
+            player.sendMessage(Util.getTextMessageWithFormat("command.iteminfo.id", it.getItem().getId(), it.getItem().getName()));
```

Comment:
Here, do not use `it.getItem().getName()`. For the human readable name use `it.getTranslation().get()`